index.html

```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *Abhishek Kuamar* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Abhishek Kumar</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="_blank"></a>
+          </p>
+          <p class="about">about you</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://github.com/abhishek44priv" target="_blank" title="First Resource">Github</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Second Resource">Email</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Third resource">StackOverflow</a>
```

Comment:
Like my other comment you'll want to make sure this third 'Useful Dev Resource' points to the link you want. Likely in this case something like `<a href="https://stackoverflow.com"`.